Cargo.toml

```diff
@@ -25,4 +25,5 @@ bitflags = "1.0.4"
 opt-level = "s"
 
 [profile.release]
-opt-level = "s"
+opt-level = 3
```

Comment:
Does this actually make the image smaller? If so, we should do the same for debug builds (and a comment explaining what's going on).

Reply:
My bad! I should've left that out. When I tested it made the image 34 bytes larger.

Reply:
I meant to put the `debug = true` there, is a comment necessary explaining why?

Reply:
Nah `debug = true` is fine as-is.

Reply:
Fixed :)